core/src/main/java/feast/core/job/dataflow/DataflowJobManager.java

```diff
@@ -183,7 +182,6 @@ public Job abortJob(Job job) {
           Strings.lenientFormat("Unable to drain job with id: %s", dataflowJobId), e);
     }
 
-    job.setStatus(JobStatus.ABORTING);
```

Comment:
This API is also called from JobService, not only JobTask. Not sure it's good idea to move it to JobTask only

Reply:
ok, I see you reusing Tasks in JobService